### ISSUE TITLE
[feature] Use focus/focal point for media in web UI

### ISSUE THREAD

**Issue by mirabilos** [superseriousbusiness/gotosocial#2593]
### Describe the bug with a clear and concise description of what the bug is. Please include screenshots of any visual issues.

When posting an image with a nōn-centred focus point, the cropped preview in the GtS web view is not appropriately shifted. (It is in FediText and, when disabling large inlines, also in Semaphore, so this is a client thing.)

### What's your GoToSocial Version?

0.13.1 git-ccecf5a

### Browser version

Firefox 115.7.0esr (Debian/amd64)

### Anything else we need to know?

[Example toot](https://toot.mirbsd.org/@mirabilos/statuses/01HNH1W3TJTSE799PN2N7K1BKK)

**Comment by tsmethurst** [comment]
Yep, this is a known thing! I'm not sure if we federate out the focal points either right now, but (if so) that should be a separate issue.

**Comment by mirabilos** [comment]
Oh, hmm. Feel free to test with the linked example toot (the focal point is at the bottom of the picture instead of in the middle where I greyed out some text).

**Comment by daenney** [comment]
Given GtS is a "backend-first" approach and we generally assume people are using a full-fledged client, I've switched this to an enhancement instead.

**Comment by mirabilos** [comment]
> Alright! We will get round to this eventually but I hope you'll understand that it's pretty low priority for now

Sure, it was just something I noticed (and which confused me).

> Given GtS is a "backend-first" approach and we generally assume people are using a full-fledged client, I've switched this to an enhancement instead.

I just used the “frontend bug” reporting category, but, sure, if it helps…
